b/BCFtools/BCFtools-1.19-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.19'
+
+homepage = 'https://www.htslib.org/'
+description = """Samtools is a suite of programs for interacting with high-throughput sequencing data.
+ BCFtools - Reading/writing BCF2/VCF/gVCF files and calling/filtering/summarising SNP and short indel sequence
+ variants"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['782b5f1bc690415192231e82213b3493b047f45e630dc8ef6f154d6126ab3e68']
+
+dependencies = [
+    ('zlib', '1.2.12'),
+    #('HTSlib', '%(version)s'),
+    ('HTSlib', '%(version)s.1'),
```

Comment:
How come that the one with `.1` is used and the other commented out?

Reply:
Normally you would use the same version for HTSlib as for BCFtools, but some minor bugs got fixed in HTSlib resulting in version 1.19.1. For BCFtools there is no 1.19.1 (yet): latest is 1.19. I left the original line commented in the EasyConfig, so when the next 1.20.x or so version is created we do not forget to revert back to '%(version)s'.